gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Jun 09 10:41:54 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
```

Comment:
In theory, do we risk breaking any end user setups by upgrading from `5.4` to `6.3`?

Reply:
Does it break your setup? Can we rely on our test suite here or? What do you foresee breaking for our end users? Are you asking, "does Gradle 6.3 build for 1.6 in an a that's incompatible with the way Gradle 5.4 builds for 1.6?"

Reply:
Phrased differently, I'd be concerned if this change breaks end user setups, while passing all tests and successfully completing all CI tasks. To fully answer you question, I'm not sure––anything is possible. However, by updating the way we build for release and by meticulously documenting the process, more light has been shed on the build process overall (I think). 

Could you offer line numbers in the `build.gradle` file that you think may affect us or our users?